web/libs/core/src/lib/utils/schema/tags.json

```diff
@@ -69,6 +69,13 @@
         "required": false,
         "default": 32
       },
+      "spectrogram": {
```

Comment:
can it be automated along with doc generation?

Reply:
that's what the fix does